scapy/route.py

```diff
@@ -154,13 +152,13 @@ def route(self,dest,verbose=None):
                 continue
             aa = atol(a)
             if aa == dst:
-                pathes.append((0xffffffffL,(LOOPBACK_NAME,a,"0.0.0.0")))
+                pathes.append((0xffffffffL,(LOOPBACK_INTERFACE,a,"0.0.0.0")))
```

Comment:
I have a question: what's the goal of this line ?
Because of it, i'm getting dupplicated routes (on windows):
- `(2130706433, 4294967295L, '0.0.0.0', <NetworkInterface Npcap Loopback Adapter {3[...]B4}>, '127.0.0.1')` (which was added because of the line)
- `(2851995648L, 4294901760L, '0.0.0.0', <NetworkInterface Npcap Loopback Adapter {3[...]B4}>, '127.0.0.1')` (which was loaded directly from the windows routes)

Is this wanted ?